train/src/train.py

```diff
@@ -275,18 +279,50 @@ def train():
         writer.add_scalar("BLEU", bleu, epoch)
         print(f"Epoch {epoch} BLEU: {bleu}")
 
+        save_best_models(epoch, model, output_dir, config, best_scores, bleu)
+        save_last_models(epoch, model, output_dir, config)
+
         scheduler.step()
```

Comment:
今回の場合問題ないけど、なんとなくモデルのsaveは最後のが良いかも。
scheduler自体もsaveしないとだめだったりしたときにズレうるので。
```suggestion
        scheduler.step()

        save_best_models(epoch, model, output_dir, config, best_scores, bleu)
        save_last_models(epoch, model, output_dir, config)
```